features/api_services_list.feature

```diff
@@ -0,0 +1,198 @@
+Feature: Services list api
+
+  @uses.config.contract_token
+  Scenario Outline: Get services list when unattached/attached
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
```

Comment:
Let's also include this API under the main api test where we import all other api endpoints:
https://github.com/canonical/ubuntu-pro-client/blob/next-v35/features/api.feature#L3